stdlib/public/Platform/tgmath.swift.gyb

```diff
@@ -122,6 +168,72 @@ func _vjpTan(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
   let value = tan(x)
   return (value, { v in v * (1 + value * value) })
 }
+
+@usableFromInline
+func _vjpAsin(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (asin(x), { v in v / sqrt(1 - x * x) })
+}
+
+@usableFromInline
+func _vjpAcos(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (acos(x), { v in -v / sqrt(1 - x * x) })
+}
+
+@usableFromInline
+func _vjpAtan(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (atan(x), { v in v / (1 + x * x) })
+}
+
+@usableFromInline
+func _vjpSinh(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (sinh(x), { v in v * cosh(x) })
+}
+
+@usableFromInline
+func _vjpCosh(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (cosh(x), { v in v * sinh(x) })
+}
+
+@usableFromInline
+func _vjpTanh(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  let value = tanh(x)
+  return (value, { v in v * (1 - value * value) })
+}
+
+@usableFromInline
+func _vjpAsinh(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (asinh(x), { v in v / sqrt(1 + x * x) })
+}
+
+@usableFromInline
+func _vjpAcosh(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (acosh(x), { v in v / sqrt(x * x - 1) })
+}
+
+@usableFromInline
+func _vjpAtanh(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (atanh(x), { v in v / (1 - x * x) })
+}
+
+@usableFromInline
+func _vjpExpm1(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (expm1(x), { v in exp(x) * v })
+}
+
+@usableFromInline
+func _vjpLog1p(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (log1p(x), { v in v / ( x + 1) })
```

Comment:
```suggestion
  return (log1p(x), { v in v / (x + 1) })
```

Reply:
done in 6b1ebf264b6801e5becf1b9f5d6206ce627af1f6